modules/tutorials/examples/logging-aggregator/main.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+echo "Installing ZooKeeper Operator"
+# tag::zk-op[]
+#stackablectl release install -i secret -i commons -i zookeeper 23.7
+stackablectl op in secret commons zookeeper
```

Comment:
I would always use the descriptive variants of the command parameters in the documentation:

```suggestion
stackablectl operator install secret commons zookeeper
```

This does not exceed the line length.